client/src/file_logger.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from ax_listener import AXFrame
+
+class FileLogger():
+
+    def __init__(self, logfile):
+        self._logger = logging.getLogger(__name__)
+        self.file_logger = logging.FileHandler(logfile)
+
+    """ Log received package to a .log file"""
+    def log_ax_frame(self, frame: AXFrame):
+        self.file_logger.handle(self._logger.makeRecord(None, logging.DEBUG, None, None, str(frame), None, None))
```

Comment:
Please add a newline at the end of your file.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795